### DISCUSSION TITLE
No logs to Visual Studio Output window when running in Release

### DISCUSSION THREAD

**Discussion by JGKle** [AvaloniaUI/Avalonia#13652] [Q&A] [answered]
I've started a new Avalonia app in Visual Studio 2022. When I run a debug build, I can see log outputs in the VS Output pane, by calling `System.Diagnostics.Debug.WriteLine("example");`. However, when I run a Release build, I cannot get any logs to appear (either this way, or using `Console.WriteLine("example");`). All of the other projects in my solution are able to log to the Output window in both Debug and Release builds. Any ideas how I can get this to work in Avalonia?

**Comment by timunie** [comment, marked as answer]
Debug.WriteLine is not available outside Debug Mode. Use Trace.WriteLine or Console.WriteLine or any Logger. https://docs.avaloniaui.net/docs/next/concepts/unhandledexceptions#logging

**Reply by JGKle** [reply]
Hi - thanks for the reply, Trace.WriteLine was the answer :)

(As mentioned in the post, Console.WriteLine doesn't work either, however)
